mesonbuild/interpreter.py

```diff
@@ -909,16 +909,28 @@ def get_name(self):
 
 class SubprojectHolder(InterpreterObject, ObjectHolder):
 
-    def __init__(self, subinterpreter):
+    def __init__(self, subinterpreter, name):
         InterpreterObject.__init__(self)
         ObjectHolder.__init__(self, subinterpreter)
+        self.name = name
         self.methods.update({'get_variable': self.get_variable_method,
```

Comment:
What happens when `get_variable()` is called on a SubprojectHolder that's not found? We should print a friendly message, and having a `run_unittests.py:FailureTest` will help us ensure that.

Reply:
`raise InterpreterException('Subproject "%s" disabled can\'t get_variable on it.' % (self.name))`. Will add a test case.

Reply:
Fixed.